libvast/test/format/arrow.cpp

```diff
@@ -56,7 +56,7 @@ TEST(arrow batch) {
   for (auto& slice : zeek_conn_log)
     writer.write(slice);
   // Cause the writer to close its current Arrow writer by switching the layout.
-  writer.layout(type{"stub", record_type{{"irrelevant", none_type{}}}});
+  REQUIRE(writer.layout(::arrow::schema({})));
```

Comment:
Have empty schemas been allowed for a while now? They used to be the main reasons for us disallowing empty record types. Maybe we need to reconsider that restriction (cc @tobim).

Reply:
I actually saw complaints about zero columns a few days ago, so I was actually surprised this works. In this particular case, it's just some sentinel to trigger a desired behavior, which is maybe not what we want. Maybe we should do an explicit:

```c++
stream->Close();
```

instead?

Reply:
Sounds good, might be worth a try.

Reply:
done